lib/order.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'menu'
+
+class Order
+
+  def initialize menu
+    @menu = menu
+    @basket = []
+  end
+
+  def add(dish, amount)
+    @basket.push([dish, amount])
+  end
+
+  def total
+    total = 0
+    @basket.each do |selection|
```

Comment:
Use reduce/inject instead of each for calculations - https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#explore-the-language-for-solutions-to-common-problems